client/src/components/BlockSearch.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+<div>
+  <q-virtual-scroll
+    class="virtual-scroll"
+    :items="results"
+  >
+    <template v-slot="{ item: user, index }">
+      <q-item
+        :key="index"
+        dense
+        clickable
+        v-if="!results[0].error"
```

Comment:
Please move "results[0].error" to computed property since it's referenced more than once.
And make sure that array is not empty before accessing "error" property